### ISSUE TITLE
Git Hub won't let my other teammates pull

### ISSUE THREAD

**Issue by MalaviRavindran** [Sta199-S18/community#27]
Both of my teammates tried to pull, and it says "please commit your changes or stash them before you can merge. error: the following untracked working tree files would be overwritten by merge"

**Comment by WalkerHarrison** [comment]
Github isn't letting you pull because the revisions you'd be receiving would conflict/overwrite something you've already saved. What you're going to want to do is commit what you have now, and *then* pull. What will result is a merge conflict, which will give you a version of the file that has both the work you did (committed) and whatever your teammates did (from the pull), surrounded by a lot of long numbers and `>>>>>>>` and the word `HEAD`. To resolve this merge conflict, pick which version of the truth you would like and then commit and push. 

So in summary:
* commit
* pull
* resolve the merge conflict (pick the edits you want and delete the arrows/long numbers)
* commit
* push

**Comment by ethann-shen** [comment]
I'm trying to commit first, but it's telling me to "use "git pull" to merge the remote branch into yours," which is confusing because I couldnt pull anything either.

**Comment by WalkerHarrison** [comment]
Hmmm. Can you type "git status" in your Terminal and tell me what it outputs (or copy paste/screenshot)?

**Comment by mine-cetinkaya-rundel** [comment]
@MalaviRavindran @ethann-shen Is the issue persisting? If so, please comment and I'll try to help resolve as well.

**Comment by ethann-shen** [comment]
This is what is popping up. 

On branch master
Your branch and 'origin/master' have diverged,
and have 3 and 15 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        lab-02-data-wrangle-viz.html
        lab-02-data-wrangle-viz.md
        lab-02-data-wrangle-viz_files/
        project.Rproj

nothing added to commit but untracked files present (use "git add" to track)

**Comment by mine-cetinkaya-rundel** [comment]
It seems like you'e not committing all of your files. Is your git pane empty completely? It should be.
